proxy/proxy.go

```diff
@@ -105,6 +105,11 @@ func (s *Handler) doHijack(message *common.HttpMessage, key string, incomingMess
 			log.WithField("error", err).Errorf("Failed to read request %s", u.Host)
 		}
 		reader.Close()
+
+		for range incomingMessages {
+			// waiting for channel to close
+		}
+		conn.Close()
```

Comment:
Decided to tune down to info, since more often then not, it will be a normal close causing it.